scripts/component.mk

```diff
@@ -0,0 +1,48 @@
+SHELL=/bin/bash
+GET_CREDS="import json, boto3.session as s; \
+           print(json.dumps(s.Session().get_credentials().get_frozen_credentials()._asdict()))"
+
+export AWS_REGION=$(shell aws configure get region)
+export AWS_ACCOUNT_ID=$(shell aws sts get-caller-identity | jq -r .Account)
+export TF_S3_BUCKET=tfstate-$(AWS_ACCOUNT_ID)
+export APP_NAME=query-service
+
+default: plan
+
+init:
+	terraform init
+
+plan:
+	make retrieve-vars
+	terraform plan -detailed-exitcode
+
+apply:
+	make retrieve-vars
+	terraform apply --backup=-
+
+retrieve-vars:
+	aws s3 cp s3://org-humancellatlas-861229788715-terraform/query-service/dev/terraform.tfvars terraform.tfvars
```

Comment:
Please move the terraform vars to Secrets Manager. This is more secure and has the added benefit of not removing the need to specify a bucket.

Here's an example https://github.com/HumanCellAtlas/logs/blob/master/Makefile#L9-L13

Reply:
👍